src/main/java/net/fabricmc/loom/configuration/providers/forge/ForgeRunsProvider.java

```diff
@@ -128,6 +128,8 @@ private String resolve(@Nullable RunConfigSettings runConfig, ConfigValue.Variab
 			// Use a set-valued multimap for deduplicating paths.
 			Multimap<String, String> modClasses = MultimapBuilder.hashKeys().linkedHashSetValues().build();
 			NamedDomainObjectContainer<ModSettings> mods = extension.getMods();
+			// Forge 49+ bootstrap-dev uses ; as a separator, instead of File.pathSeparator
+			String separator = extension.getForgeProvider().getVersion().getMajorVersion() >= Constants.Forge.MIN_BOOTSTRAP_DEV_VERSION ? ";" : File.pathSeparator;
```

Comment:
Note for myself: Unfortunately this needs a check for the bootstrap too since Forge changed it back to File.pathSeparator.

Reply:
This is now handled!